src/Container/NotFoundHandlerFactory.php

```diff
@@ -1,7 +1,7 @@
 <?php
 /**
  * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
- * @copyright Copyright (c) 2016-2017 Zend Technologies USA Inc. (https://www.zend.com)
+ * @copyright Copyright (c) 2018 Zend Technologies USA Inc. (https://www.zend.com)
```

Comment:
Should we keep year range here?

Reply:
No, because in this case, it's referencing a brand new class, so this factory is technically new.